tests/index.test.js

```diff
@@ -29,6 +29,12 @@ describe('jest-fail-on-console', () => {
     expect(stderr).toEqual(expect.stringContaining(passString('error-disabled')))
   })
 
+  it.only('does not error when console.error() is called and skip test returns true', async () => {
```

Comment:
skip should be removed 😅

Reply:
whoops, my bad 😅 got too used to linters warning me about this 3175ba9ac6f4b1844d830df4054539643a27ac49

Reply:
Yes I understand, same here!